src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10374,14 +10381,16 @@ private TupleExpressionSyntax ParseTupleExpressionTail(SyntaxToken openParen, Ar
                     list.Add(arg);
                 }
 
-                var closeParen = this.EatToken(SyntaxKind.CloseParenToken);
-                var result = _syntaxFactory.TupleExpression(openParen, list, closeParen);
-
                 if (list.Count < 2)
                 {
-                    result = this.AddError(result, ErrorCode.ERR_TupleTooFewElements);
+                    list.AddSeparator(SyntaxFactory.MissingToken(SyntaxKind.CommaToken));
+                    var missingExpression = this.AddError(this.CreateMissingIdentifierName(), ErrorCode.ERR_TupleTooFewElements);
+                    list.Add(_syntaxFactory.Argument(nameColon: null, refOrOutKeyword: default(SyntaxToken), expression: missingExpression));
```

Comment:
Yes, it is valid.
Test `TupleExpressionWithTooFewElements` has tuple expression `(Alice: 1)` and test `TupleTypeWithTooFewElements` has tuple type `(int a)`.